### PR TITLE
2023 Annual Report on Pegasus

### PR DESCRIPTION
**Pull request by hannahbergam**
This is the phase 1 of our 2023 annual report page. I was able to reuse a lot from the previous two years of editions of this page (see [2021](https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites.v3/code.org/public/about/2021.md.partial) and [2022](https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites.v3/code.org/public/about/2022.md.partial)).

I added the backup video to the Amazon [s3 bucket here](https://us-east-1.console.aws.amazon.com/s3/buckets/videos.code.org?region=us-east-1&bucketType=general&tab=objects) and uploaded the new pdf to live where the 2021 pdf lives. 

| From Figma | This PR |
:-------------------------:|:-------------------------:
Screenshot 2024-05-20 at 1 54 39 PM | 

## Links

- spec: []()
- jira ticket: https://codedotorg.atlassian.net/browse/ACQ-1881

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: pegasus/sites.v3/code.org/views/vid_hadi_2023.erb [hunk 1]
```diff
@@ -0,0 +1 @@
+<%=view :display_video_thumbnail, id: "2023_Hadi_Annual_Report", video_code: "XjCGHs3J3mk", play_button: 'center', letterbox: 'false' %>
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/views/vid_hadi_2023.erb]: Here's how to add the download path:
```suggestion
<%=view :display_video_thumbnail, id: "2023_Hadi_Annual_Report", video_code: "XjCGHs3J3mk", play_button: 'center', letterbox: 'false', download_path: "//videos.code.org/2023_Hadi_Annual_Report.mp4" %>
```

**Review by kelbyhawn — COMMENTED** [review]
Needs one minor update to add the `download_path` for Hadi's video, but looks good otherwise!

File: pegasus/sites.v3/code.org/views/vid_hadi_2023.erb (hunk 1 above)
Comment (hannahbergam) [reply pegasus/sites.v3/code.org/views/vid_hadi_2023.erb]: Added and tested! Thank you!

**Review by kelbyhawn — APPROVED** [review]
Woo hoo, thanks for adding the download video!

**Pull request merged**